mycroft/skills/intent_services/adapt_service.py

```
@@ -13,10 +13,10 @@
 # limitations under the License.
 #
 """An intent parsing service using the Adapt parser."""
-from adapt.context import ContextManagerFrame
-from adapt.engine import IntentDeterminationEngine
+from ovos_adapt.context import ContextManagerFrame
+from ovos_adapt.engine import IntentDeterminationEngine
 from ovos_workshop.intents import IntentBuilder, Intent
-from ovos_core.intent_services.adapt_service import ContextManager, AdaptService
+from ovos_adapt.opm import ContextManager, AdaptPipeline as AdaptService
```

Ruff

`ovos_adapt.opm.ContextManager` imported but unused

Remove unused import

(F401)